plugins/modules/aci_epg.py

```diff
@@ -77,7 +77,7 @@
     description:
     - Use C(true) to create uSeg EPG and C(false) is used to create Application EPG.
     type: str
-    choices: [ 'yes', 'no' ]
+    choices: [ true, false]
```

Comment:
Will this module still work? The argument_spec still requires `useg=dict(type="str", choices=["yes", "no"])`, so would expect the code to be adjusted to accept bool input and map to string that should be configured on APIC.

Reply:
A example of already existing function is found in M(aci_aa_user).

```
clear_password_history = aci.boolean(module.params.get("clear_password_history"), "yes", "no")

Reply:
checked and fixed